saleor/channel/migrations/0007_order_settings_per_channel.py

```diff
@@ -7,14 +7,20 @@ def set_order_settings(apps, schema_editor):
 
     site_settings = SiteSettings.objects.first()
 
-    Channel.objects.update(
-        automatically_confirm_all_new_orders=(
-            site_settings.automatically_confirm_all_new_orders
-        ),
-        automatically_fulfill_non_shippable_gift_card=(
-            site_settings.automatically_fulfill_non_shippable_gift_card
-        ),
-    )
+    if site_settings:
```

Comment:
Did you check if it isn't a problem with missing or wrong dependency? It's kind of weird that the `SiteSettings` object doesn't exist here.

Reply:
Yes, after the issue was reported, I did recheck all migrations and flow. Migrations in this PR are dependent on `site 0002`. It happend only with `--reuse-db` when running tests.